deployments/data101/config/common.yaml

```diff
@@ -19,6 +19,20 @@ jupyterhub:
         hub.jupyter.org/pool-name: core-pool-2023-07-11
 
   hub:
+    config:
+    loadRoles:
+      # Data 101, Fall 2023
+      course-staff-1528038:
+        description: Enable course staff to view and access servers.
+        # this role provides permissions to...
+        scopes:
+          - admin-ui
+          - list:users!group=course::1527426
+          - admin:servers!group=course::1527426
+          - access:servers!group=course::1527426
+        # this role will be assigned to...
+        groups:
+          - course::1528038::group::Admins
```

Comment:
Looks good, as long as `course::1528038::group::Admins` exists. Perhaps double-check that the "Admins" bCourses group exists for the course?

Reply:
@ryanlovett I can't see a bcourses admin group for Data 101. I see that 9 folks are affiliated as a teacher. Does that mean the admin group is formed for this course?

Reply:
I will merge this PR and request the instructor to create a bcourses admin group (if needed).